Code/GraphMol/SynthonSpaceSearch/fingerprint_search_catch_tests.cpp

```diff
@@ -110,6 +110,15 @@ TEST_CASE("FP Small tests") {
       resSmis.insert(MolToSmiles(*r));
     }
 
+    // test with callback version
+    std::set<std::string> cbSmis;
+    auto cb = [&cbSmis](std::vector<std::unique_ptr<ROMol>> &results) {
+      for (auto &r : results) cbSmis.insert(MolToSmiles(*r));
```

Comment:
We don't do one liners; please put curly brackets around things:

```suggestion
      for (const auto &r : results) {
        cbSmis.insert(MolToSmiles(*r));
      }
```